pkgs/tools/security/aflplusplus/qemu.nix

```diff
@@ -24,8 +24,8 @@ stdenv.mkDerivation {
   src = fetchFromGitHub {
     owner = "AFLplusplus";
     repo = "qemuafl";
-    rev = "a1321713c7502c152dd7527555e0f8a800d55225";
-    sha256 = "sha256-HLlOHqT2vrHjHyu4n83IzVzKv9ErinephLLev1E10nM=";
+    rev = "40033af00c4c5de172ed4fe60c21b9edbd2c189d";
```

Comment:
In the next update of AFLplusplus, how would we be able to know what revision to use? Is this a submodule or something alike?

Reply:
I think this is this: https://github.com/AFLplusplus/AFLplusplus/commits/stable/qemu_mode/QEMUAFL_VERSION

Reply:
This commit https://github.com/AFLplusplus/AFLplusplus/commit/07bc202e0ad940e0cc7c8770f69ceb32ed851384 sets it to `40033af`

Reply:
> I think this is this: [AFLplusplus/AFLplusplus@`stable`/qemu_mode/QEMUAFL_VERSION (commits)](https://github.com/AFLplusplus/AFLplusplus/commits/stable/qemu_mode/QEMUAFL_VERSION)

@Sigmanificient this is a great link to put in a comment near that revision. Could you please do that? Besides that this PR can be merged IMO.